R/util-render_rmd.R

```diff
@@ -0,0 +1,24 @@
+#' Custom Rmarkdown render function
+#'
+#' Rmarkdown render function that defaults to rendering intermediate rmd files in a temporary directory
+#'
+#' @param input path to the template Rmd file
+#' @param output_file path to the loacation where output will be saved
+#' @param params list of params to pass to the template Rmd file
```

Comment:
```suggestion
#' @param strInputPath `string` or `fs_path` Path to the template `Rmd` file.
#' @param strOutputPath `string` or `fs_path` Path to the location where output will be saved.
#' @param lParams `list` List of parameters to pass to the template `Rmd` file.
```

Since we're exporting this, we should follow our conventions. I'll also add changes for the places where these are used.

Reply:
...but see above, I think probably more should move into this.